lib/kitchen/directions/bake_notes/bake_numbered_note.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Kitchen
+  module Directions
+    module BakeNumberedNote
+      def self.v1(note:)
+        note.wrap_children(class: 'os-note-body')
+
+        chapter_count = note.ancestor(:chapter).count_in(:book)
```

Comment:
why not `note.count_in(:book)`?

Reply:
The numbering of these notes is, for example, 2.1 for the first note in chapter 2. So this isn't counting how many notes are in the book, but instead getting the count of the note's parent chapter.

Reply:
oh duh